src/Microsoft.IdentityModel.Tokens/Encryption/SymmetricKeyWrapProvider.cs

```diff
@@ -180,7 +180,7 @@ protected virtual SymmetricAlgorithm GetSymmetricAlgorithm(SecurityKey key, stri
             {
                 // Create the AES provider
                 SymmetricAlgorithm symmetricAlgorithm = Aes.Create();
-                symmetricAlgorithm.Mode = CipherMode.ECB;
+                symmetricAlgorithm.Mode = CipherMode.CBC;
```

Comment:
I don't think these are backward compatible, so this would break any tokens that are in flight.
So we can't do this.

Reply:
IMHO, you should consider adopting this breaking change for two reasons:
  - While using it with random content - such as wrapped keys - is not as problematic as with non-random plaintext, ECB is an unsafe AES mode: no one should ever use it.
  - It's not standard: the JWA/JWE specs only list CBC as an allowed AES mode: https://datatracker.ietf.org/doc/html/rfc7518#section-5.1 / https://datatracker.ietf.org/doc/html/rfc7516#appendix-A.3

Reply:
@kevinchalet i agree with you, i am think hard about how not to break tokens in flight. This is a real issue.
I agree ECB is less secure than CBC, but for small number of bytes it is not a significant as larger data.